userspace/falco/stats_writer.cpp

```diff
@@ -329,11 +343,13 @@ void stats_writer::collector::get_metrics_output_fields_additional(
 					{
 						output_fields["scap.evts_rate_sec"] = (double)(0);
 					}
-					output_fields["scap.n_evts_prev"] = m_last_n_evts;
 				}
-				else if (strncmp(stats_v2[stat].name, "n_drops", 7) == 0)
+				/* Always send high level n_drops related fields, even if zero. */
+				else if (strncmp(stats_v2[stat].name, "n_drops", 8) == 0) // exact not prefix match here
```

Comment:
In addition @jasondellaluce I think this also would have been a bug and had matched also the other `n_drops_*` prefixed metrics and given the wrong calculations.